### PR TITLE
Github actions for more connectivity!

### LINKED ISSUE #1088: Github actions for more connectivity!
**We can implement github workflows for the following things:**

1. Auto comment on issue **Closed** and **Opened**
2. Auto comment on **PR mergs**
3. Auto comment on **PR raises**

Using this workflows makes you'r repo more interactive ! @ajay-dhangar 

_**i'm expecting a level3 with gssoc-ext and hacktoberfest-accepted label**_

### PR DESCRIPTION
**Pull request by smog-root**
Closes #1088 

1. Auto comment on issue **Closed** and **Opened**
2. Auto comment on **PR mergs**
3. Auto comment on **PR raises**

Using this workflows makes you'r repo more interactive ! @ajay-dhangar 

**Proof**:

**Issue open and close:**

Screenshot 2024-10-21 152058
<br>

**PR raise and  merged:**
Screenshot 2024-10-21 160211

_**Note**_: Add level with gssoc-ext and hacktober-accepted labels! assign me to this PR !!

### CODE REVIEW

**Comment by smog-root** [comment]
Add level2 cause i implemented 3 workflows! and updated a old workflow!  total 4 workflows!!

@ajay-dhangar

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Can you recheck your code because it's content not related to us may be refreshing to the repo

**Comment by smog-root** [comment]
What are the changes i should do in workflows??

File: .github/workflows/pr_raise.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  comment:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - name: Add Comment to Pull Request
+      run: |
+        COMMENT=$(cat <<EOF
+        {
+          "body": "Thank you for submitting your pull request! 🙌 We'll review it as soon as possible. In the meantime, please ensure that your changes align with our chaotic [CONTRIBUTING.md](https://github.com/vansh-codes/ChaosWeb/blob/main/CONTRIBUTING). If there are any specific instructions or feedback regarding your PR, we'll provide them here. Thanks again for your contribution! 😊"
```
Comment (ajay-dhangar) [inline .github/workflows/pr_raise.yml]: checkout this content

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
We don't like to use other repos actions because our first priority is to use actions based on our content, codes, and our tech stack.

File: .github/workflows/pr_raise.yml (hunk 1 above)
Comment (smog-root) [reply .github/workflows/pr_raise.yml]: Hoo. Shit it was miss placed. I will change it!!

**Comment by smog-root** [comment]
Changed!!

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**